packages/shared/src/components/profile/ProfileSettingsMenu.tsx

```diff
@@ -0,0 +1,135 @@
+import React, { ReactElement, useMemo } from 'react';
+import {
+  AddUserIcon,
+  BellIcon,
+  CardIcon,
+  EditIcon,
+  LockIcon,
+  MenuIcon,
+  DevCardIcon,
+  ExitIcon,
+  EmbedIcon,
+  LinkIcon,
+  DocsIcon,
+  TerminalIcon,
+  FeedbackIcon,
+  HammerIcon,
+} from '../icons';
+import { NavDrawer } from '../drawers/NavDrawer';
+import {
+  docs,
+  feedback,
+  privacyPolicy,
+  termsOfService,
+} from '../../lib/constants';
+import { useLazyModal } from '../../hooks/useLazyModal';
+import { LazyModal } from '../modals/common/types';
+import { LazyModalType } from '../modals/common';
+
+const createMenuItems = (
+  logout: () => Promise<void>,
+  openModal: (data: LazyModalType<LazyModal>) => void,
+) => {
+  return [
+    {
+      label: 'Profile',
+      isHeader: true,
+    },
+    { label: 'Edit profile', icon: <EditIcon />, href: '/account/profile' },
+    { label: 'Invite friends', icon: <AddUserIcon />, href: '/account/invite' },
+    { label: 'Devcard', icon: <DevCardIcon />, href: '/devcard' },
+    { label: 'Logout', icon: <ExitIcon />, onClick: logout },
+    {
+      label: 'Manage',
+      isHeader: true,
+    },
+    {
+      label: 'Customize',
+      icon: <CardIcon />,
+      onClick: () => openModal({ type: LazyModal.UserSettings }),
+    },
+    { label: 'Security', icon: <LockIcon />, href: '/account/security' },
+    {
+      label: 'Notifications',
+      icon: <BellIcon />,
+      href: '/account/notifications',
+    },
+    {
+      label: 'Other settings',
+      icon: <MenuIcon className="rotate-90" />,
+      href: '/account/others',
+    },
+    {
+      label: 'Contribute',
+      isHeader: true,
+    },
+    {
+      label: 'Submit article',
+      icon: <LinkIcon />,
+      onClick: () => openModal({ type: LazyModal.SubmitArticle }),
+    },
+    {
+      label: 'Suggest new source',
+      icon: <EmbedIcon />,
+      onClick: () => openModal({ type: LazyModal.NewSource }),
+    },
+    {
+      label: 'Support',
+      isHeader: true,
+    },
+    { label: 'Docs', icon: <DocsIcon />, href: docs, target: '_blank' },
```

Comment:
Here we'll need the default rel additon (*bascially everywhere you go with `_blank` or that's not app.daily.dev.